src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -74,7 +75,16 @@ class AutomaticSpeechRecognitionPipeline(Pipeline):
     to support multiple audio formats
     """
 
-    def __init__(self, feature_extractor: Union["SequenceFeatureExtractor", str], *args, **kwargs):
+    def __init__(
+        self,
+        feature_extractor: Union["SequenceFeatureExtractor", str],
+        chunking=False,
+        chunk_len=128080,
+        chunk_start_padding_len=16160,
+        chunk_end_padding_len=16160,
```

Comment:
This roughly corresponds to 8-second chunks with 1-sec padding on both sides.
The values are so "specific" to avoid border artifacts after convolutional feature extraction inside the Wav2Vec2 models. Basically, the these input lengths are evenly divided by `conv_kernel` size at each feature extractor layer.

Reply:
Would be great to leave this as a comment

Reply:
Ok for me!